src/main/scala/pipelines/images/voc/VOCSIFTFisher.scala

```diff
@@ -67,10 +69,9 @@ object VOCSIFTFisher extends Serializable with Logging {
           csvread(new File(conf.gmmWtsFile.get)).toDenseVector)
         pcaFeaturizer andThen FisherVector(gmm)
       case None =>
-        val fisherVector = pcaFeaturizer andThen
-            ColumnSampler(numGMMSamplesPerImage) andThen
-            (GMMFisherVectorEstimator(conf.vocabSize), trainingData)
-        pcaFeaturizer andThen fisherVector.fittedTransformer
+        val sampler = ColumnSampler(numGMMSamplesPerImage).toPipeline
+        val fisherVector = GMMFisherVectorEstimator(conf.vocabSize) withData (sampler(pcaFeaturizer(trainingData)))
+        pcaFeaturizer
```

Comment:
Very good catch - may also explain some of my integration test woes. Will fix this and re-run.